.github/workflows/pr.yaml

```diff
@@ -2,19 +2,10 @@ name: Pull Request
 
 
 on:
-  pull_request:
-    paths:
-      - ".github/workflows/pr.yaml"
-      - ".github/workflows/tf-plan.yaml"
-      - "client/**"
-      - "server/**"
-      - "terraform/**"
-      - "!terraform/sandbox/**"
-      - "Dockerfile"
-      - "!**/README.md"
+  workflow_dispatch:
```

Comment:
This shouldn't be changing, this makes it a manual workflow and not triggered on PR

Reply:
Done.